packages/block-library/src/post-title/edit.js

```diff
@@ -12,7 +12,7 @@ import {
 	BlockControls,
 	InspectorControls,
 	useBlockProps,
-	PlainText,
+	RichText,
```

Comment:
Hi @kapilpaul, thanks for attempting this PR as a first contribution 🎉 

However, simply changing the block's element from PlainText to RichText has a number of unexpected effects in other places that use the post title, as it currently is not expected to contain rich text and the new HTML added by the RichText is displayed as content, like in the example below:

https://user-images.githubusercontent.com/27339341/116901589-67885280-ac3a-11eb-8606-c26f4ea2f994.mov

Reply:
Hi @priethor Thanks for reviewing it. I understand this will cause a lot of issues. So in that case, how can I achieve this feature? Do you have anything in mind? Can you please guide me?

Reply:
Achieving fine-grained styling options that require markup inside the post title content would require quite some architectural changes, so a better approach would be adding styles that affect the whole block rather than the content in a similar way Font Size and Line Height work.

Users won't be able to make "Hello" bold and "world" italic, for example, which I would say is a very specific and not-so-common use case, but the whole block would be bold and/or italic. As these would be saved as block attributes and not inside the post title as markup, other uses of the post title like `get_the_title()` won't be impacted.

Reply:
Exactly that comes to my mind. I will look deeper into it. Thanks for the guidance. I will work on it :)